player/ad-event-verification-omsdk/index.html

```diff
@@ -70,19 +70,10 @@ <h2>
         </h2>
     </div>
     <div class="w-100 my-4">
-        <p>
-            1. Acquire the OM SDK files from the
-            <a href="https://tools.iabtechlab.com/omsdk">
-                IAB Tech Lab Tools Portal
-            </a>
-        </p>
         <p class="mb-3">
-            2. Include the following scripts in your page
+            1. Include the following scripts in your page
         </p>
         <ul class="ml-4">
```

Comment:
I think we should get rid of this `<ul>` here as it now has only a single list entry and instead consolidate with line 74 🙂